src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -558,17 +566,51 @@ def transform_manifest(self, format: str, filters: JSON):
         source_filter = [field_path_prefix + '.' + field_name
                          for field_path_prefix, field_mapping in manifest_config.items()
                          for field_name in field_mapping.values()]
-        es_search = self.create_request(filters,
-                                        self.es_client,
-                                        request_config,
-                                        post_filter=False,
-                                        source_filter=source_filter,
-                                        enable_aggregation=False)
+        es_search = self.es_search_request(filters, request_config, source_filter)
 
         manifest = ManifestResponse(es_search, manifest_config, request_config['translation'], format)
 
         return manifest.return_response()
 
+    def es_search_request(self,
```

Comment:
I don't think this method helps. It only adds yet another method with too large a signature. And it reorders arguments so as to create more confusion.